app/views/simulator/edit.html.erb

```diff
@@ -106,7 +106,7 @@
       <li class="dropdown pull-right">
         <% if user_signed_in? %>
           <a href="#" class="" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false"><%= current_user.name %><span class="caret"></span></a>
-          <ul class="dropdown-menu">
+          <ul class="dropdown-menu" style="min-width: 110px !important;">
```

Comment:
can we avoid inline styles?

Reply:
I guess this one might be an exceptional case to use inline style

Reply:
the !important surely need not be there, i have removed it

Reply:
ok I see, done it.